diagnostic.md

```diff
@@ -14,7 +17,8 @@ list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
 <!-- your answer starts here -->
-
+$('#li').show();
```

Comment:
This will select an element with an `id` of `li` and then invoke `.show()` on it.